mail-server/users.nix

```diff
@@ -55,16 +55,16 @@ let
     # is null, remove the file.
     ${lib.concatMapStringsSep "\n" ({ name, sieveScript }:
       if lib.isString sieveScript then ''
-        cat << EOF > "/var/sieve/${name}.sieve"
+        cat << EOF > "/var/sieve/${name}/default.sieve"
         ${sieveScript}
         EOF
-        chown "${name}:${vmailGroupName}" "/var/sieve/${name}.sieve"
+        chown "${name}:${vmailGroupName}" "/var/sieve/${name}/default.sieve"
       '' else ''
-        if (test -f "/var/sieve/${name}.sieve"); then
-          rm "/var/sieve/${name}.sieve"
+        if (test -f "/var/sieve/${name}/default.sieve"); then
+          rm "/var/sieve/${name}/default.sieve"
```

Comment:
I guess here we can simply remove the folder now

Reply:
The folder may still contain user's scripts uploaded via ManageSieve, so let's just keep it.